hardcore_reset.py

```diff
@@ -29,6 +30,32 @@ def hard_reset(skip = False):
     if skip == False:
         print('Progress deleted!\n')
         print('Creating space for a new database....')
+    c.execute("CREATE TABLE `inventory` ('id' INTEGER NOT NULL, \
+	'name' TEXT NOT NULL, \
+	'000' INTEGER NOT NULL DEFAULT 0, \
+	'001' INTEGER NOT NULL DEFAULT 0, \
+	'002' INTEGER NOT NULL DEFAULT 0, \
+	'003' INTEGER NOT NULL DEFAULT 0, \
+	'004' INTEGER NOT NULL DEFAULT 0, \
+	'005' INTEGER NOT NULL DEFAULT 0, \
+	'006' INTEGER NOT NULL DEFAULT 0, \
+	'007' INTEGER NOT NULL DEFAULT 0, \
+	'008' INTEGER NOT NULL DEFAULT 0, \
+	'009' INTEGER NOT NULL DEFAULT 0, \
+	'010' INTEGER NOT NULL DEFAULT 0, \
+	'011' INTEGER NOT NULL DEFAULT 0, \
+	'012' INTEGER NOT NULL DEFAULT 0, \
+	'013' INTEGER NOT NULL DEFAULT 0, \
+	'014' INTEGER NOT NULL DEFAULT 0, \
+	'015' INTEGER NOT NULL DEFAULT 0, \
+	'100' INTEGER NOT NULL DEFAULT 0, \
+	'101' INTEGER NOT NULL DEFAULT 0, \
+	'102' INTEGER NOT NULL DEFAULT 0, \
+	'103' INTEGER NOT NULL DEFAULT 0, \
+	'104' INTEGER NOT NULL DEFAULT 0, \
+	'105' INTEGER NOT NULL DEFAULT 0, \
+	'106' INTEGER NOT NULL DEFAULT 0, PRIMARY KEY('id'));")

```

Comment:
erm what?

Reply:
That's strange.

...I don't have that ^M right there. 🤔

Reply:
well it is there. maybe its an escape code.. like backspace sometimes is ^H or ^C is a trigger to interrupt processes.. like CTRL+<letter>. anyways, itll have to DIEEEE!! as it is a syntax error

Reply:
The Evil One doesn't appear to complain avout it.

Reply:
AAAGH AM I STILL THERET WHYYY

Reply:
oh come on why do you need a second account?

Reply:
I do because it's more convenient to be able to pull directly from the virtual machine and use whatever is handed to me.

And no, I ain't givin' Herr Rucan nor mah Buddy1913 access to meh main account!